generator/src/Command/ScrapeCommand.php

```diff
@@ -0,0 +1,246 @@
+<?php
+declare(strict_types=1);
+
+namespace MongoDB\CodeGenerator\Command;
+
+use InvalidArgumentException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\Yaml\Yaml;
+
+use function array_column;
+use function array_combine;
+use function array_key_exists;
+use function array_shift;
+use function assert;
+use function explode;
+use function file_exists;
+use function file_get_contents;
+use function file_put_contents;
+use function htmlspecialchars_decode;
+use function ksort;
+use function mkdir;
+use function sort;
+use function str_contains;
+use function str_ends_with;
+use function str_replace;
+use function strip_tags;
+use function trim;
+use function var_export;
+
+use const PHP_EOL;
+
+final class ScrapeCommand extends Command
```

Comment:
I'll probably remove this command before merging. The google sheet is a step for generating hundreds of Yaml file. But it is not suitable for day-to-day maintenance and versioning.

Reply:
It's all done by hand. Unless you use AI, I don't see how you can recreate this file simply to detect changes.

The form in which things are documented varies greatly and some informations where extracted from the server source code. I'd better maintain Yaml files and get notified of changes.